### PR TITLE
Feat/lesq 126/slide deck styling

### PR DESCRIPTION
**Pull request by kimon-satan**
I want to see the slide deck with the new styling

## Acceptance Criteria:

- [x]  “Slide deck” title
- [x]  Button to download
- [x]  Styled frame around the google slides component

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1773--oak-web-application.netlify.thenational.academy/beta/teachers/programmes/combined-science-secondary-ks4-higher/units/cells-biology-3p8njnj/lessons/prokaryotic-and-eukaryotic-cells-b9qqye

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

**Review by jack-skerrett-oak — COMMENTED** [review]
Looks good across browsers/devices!

Curious about the border.. the lil' "hand drawn" notch isn't there. Also it feels a bit "square". I'm not sure if that is just our poor implementation of the brush borders currently (most of the other places it appears it is relatively square). Just thought I would flag.

**Review by JBR90 — APPROVED** [review]
LGTM - Just need to add the box border prop

File: src/components/pages/TeachersLessonOverview/OverviewPresentation.tsx [hunk 1]
```diff
@@ -20,19 +19,21 @@ const OverviewPresentation: FC<OverviewPresentationProps> = ({
...
-          width="100%"
-          height="100%"
-          // We know the google slides aren't accessible.
-          className="pa11y-ignore"
-          data-testid="overview-presentation"
-        />
-      </AspectRatio>
-    </OverviewAssetWrap>
+    <AspectRatio ratio={isWorksheetPortrait ? "2:3" : "16:9"}>
+      <BoxBorders />
```
[the remark this answers is not in the document]
Comment (kimon-satan) [reply src/components/pages/TeachersLessonOverview/OverviewPresentation.tsx:23]: Just checking do you mean this ?
image

[the remark this answers is not in the document]
Comment (JBR90) [reply src/components/pages/TeachersLessonOverview/OverviewPresentation.tsx:23]: yes but in the designs it's top right

image

**Pull request merged**